docs/en/enterprise-edition/policy-reference/secrets-policies/secrets-policy-index/git-secrets-256.adoc

```diff
@@ -0,0 +1,42 @@
+== Rails Master Key detected in code
+
+
+=== Policy Details
+
+[width=45%]
+[cols="1,1"]
+|===
+|Prisma Cloud Policy ID
+|90f9b054-003f-4e00-85b2-fadec8b66dc5
+
+|Checkov Check ID
+|CKV_SECRET_256
+
+|Severity
+|MEDIUM
+
+|Subtype
+|build
+
+|Frameworks
+|Git
+
+|===
+
+
+=== Description
+
+The Rails Master Key is a highly sensitive credential used in Ruby on Rails applications for encrypting sensitive data, such as session cookies and database credentials. If your Rails Master Key is exposed, attackers could potentially decrypt sensitive information, hijack user sessions, or gain unauthorized access to your application and its data.
+
+=== Fix - Buildtime
+
+*Rails Master Key*
+
+**Never store your Rails Master Key directly in your code repository.** Instead, use environment variables or a secure credentials management system to store and manage the key. 
+
+If you have accidentally committed your Rails Master Key to your code repository, take immediate action:
+
+1. **Invalidate the Exposed Key:** Change your Rails Master Key immediately. Refer to the Ruby on Rails security documentation for instructions on how to generate and set a new Master Key.
```

Comment:
**Invalidate the Exposed Key**: Rotate your Rails Master Key immediately. Refer to the Ruby on Rails security documentation for instructions on how to generate and set a new Master Key.
@tsmithv11  - no link to docs?